TLSharp.Core/TelegramClient.cs

```diff
@@ -134,8 +136,9 @@ private async Task ReconnectToDcAsync(int dcId, CancellationToken token = defaul
             
             var dataCenter = new DataCenter (dcId, dc.IpAddress, dc.Port);
 
-            transport = new TcpTransport(dc.IpAddress, dc.Port, handler);
+            transport = new TcpTransport(dc.IpAddress, dc.Port, handler); // this will be gone after fix #937
```

Comment:
if you keep this line after fix 937, it will be useless because the connectionAsync on line 140 will cause  a recreation of the TcpTransport

Reply:
Right so then this has to be fixed in the PR#937, not in the same commit that does the MediaOnly filter.

Reply:
ok then
let's go ahead.

nothing personal, ok, i am a very direct and at the same time sincere person. 

cheers